script/script.js

```diff
@@ -21,3 +21,125 @@ hamburger.addEventListener('click', mobileMenu);
 menuItems.forEach(
   (menuItem) => menuItem.addEventListener('click', mobileMenu),
 );
+
```

Comment:
- [ ] For the menu implementation I think it would be great if you add home link in your about page so the user can back to the home page and also according to the requirements to follow the same design and it is shown there to add a link to the home and that's good for user experinence 🙏🏻🙏🏻🔥🔥🔥

- [ ] Kindly add in both mobile menu in the mobile version and in the  desktop version 🔥🔥🔥